sklearn/tests/test_config.py

```diff
@@ -0,0 +1,26 @@
+from sklearn import get_config, set_config
+from sklearn.utils.testing import assert_equal
+
+
+def test_set_config():
+    assert_equal(get_config(), {'assume_finite': False})
+
+    # Not using as a context manager affects nothing
+    set_config(assume_finite=True)
```

Comment:
Oh, my confusion: numpy's `seterr` changes things globally. numpy's `errstate` changes context.

* I'll rename this to `config_state` or `config_context`. Which?
* Do you think I should provide `set_config` as well?